container-chains/templates/simple/runtime/src/lib.rs

```diff
@@ -902,7 +916,7 @@ impl_runtime_apis! {
 }
 
 struct CheckInherents;
-
+#[allow(deprecated)]
```

Comment:
TODO: re-check. The warning suggests removing `CheckInherents` struct. We should be careful about the implications of this.